packages/as-types/assembly/argument.ts

```diff
@@ -42,22 +42,26 @@ export class Args {
   private serialized: StaticArray<u8> = new StaticArray<u8>(0);
 
```

Comment:
Missing @remarks in class description ?
Maybe use @link to talk about specific functions from the class also.